apps/astarte_pairing/lib/astarte_pairing/queries.ex

```diff
@@ -223,6 +224,32 @@ defmodule Astarte.Pairing.Queries do
     {:ok, count}
   end
 
+  def create_ownership_voucher(realm_name, voucher_blob, pkey_blob) do
+    keyspace_name = Realm.keyspace_name(realm_name)
+    opts = [prefix: keyspace_name, consistency: Consistency.device_info(:write)]
+
+    timestamp_ms =
```

Comment:
this is no longer a timestamp and neither is it in ms


```suggestion
    expiry =
```